pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -10715,6 +10715,63 @@
 
   language_label: "Language"
 
+  hoc_homepage:
+    what_is_hoc:
+      heading: "What is the Hour of Code?"
+      desc: "The Hour of Code is worldwide movement that aims to introduce millions of students to computer science through one-hour coding activities. Through Hour of Code, we aim to demystify coding and show that anyone can learn the basics, inspiring future interest in computer science."
+    host:
+      heading: "Host an Hour of Code"
+      desc: "Your class or school can join millions around the world doing the Hour of Code! Registration for the annual Computer Science Education Week celebration begins in September. Though, Hour of Code is available year-round. Register today!"
+      button: "Register your event"
+    resources:
+      heading: "All the resources you need for your Hour of Code"
+      desc: "Bringing Hour of Code to your classroom or school is easy! Follow our step-by-step guides for preparing for, running, and celebrating your event. Check out our resources for everything you'll need!"
```

Comment:
Okay I'm also feeling weird about the inconsistencies between referring to it as "Hour of Code" vs using 'the' before (which it seems like they haven't chosen one or the other)

Reply:
Oh good point…I'm not sure it's worth delaying things further though 🤔 @bethanyaconnor do you think this is something we should bring to the marketing team?

Reply:
Ugh I'm torn -- on one hand, I definitely see it but on the other hand, I'm weary of randomizing this work. I also wonder if we're noticing this because we're literally combing through every single line in a way most people wouldn't. 

Given the timelines, my gut says to just go with what's in the Figma. But I'm definitely open to push back. cc @dmcavoy in case you have thoughts.

Reply:
I think I agree w/ leaving it as-is. People mostly skim copy anyway 😞

Reply:
Thanks for calling out team. I think its fine to go with the copy that marketing provided

Reply:
All good then! In the future I'll read only for typos and not worry too much about content:)